app/views/observer/show_user.html.erb

```diff
@@ -61,7 +61,11 @@
 
   # These are roughly in decreasing order of importance.
   rows = []
-  for field in SiteData::ALL_FIELDS - [:users]
+  (SiteData::ALL_FIELDS -
+   SiteData::SITE_WIDE_FIELDS -
+   # omit sequence stuff because it has no weight
+   [:sequences, :sequenced_observations]).
```

Comment:
I also think that showing these in the User Summary is more likely to trigger an argument about the Contributions algorithm. I.e., that it should include sequences. 
(I would favor including sequences, but coding it is not a priority.)